src/App.jsx

```diff
@@ -13,34 +14,83 @@ export const goodsFromServer = [
   'Jam',
   'Garlic',
 ];
+const SORT_FIELD_ALPHABET = 'alphabet';
+const SORT_FIELD_LENGTH = 'length';
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+export const App = () => {
+  const [sortField, setSortField] = useState('');
```

Comment:
The initial state of `sortField` is set to an empty string. It might be more intuitive to set it to `null` or a specific default value to clearly indicate that no sorting is applied initially. This can help avoid potential confusion when checking the state elsewhere in the code.